doc-content/enterprise-only/admin-and-config/managing-business-central-githook-notifications-exit-codes-types-con.adoc

```diff
@@ -1,19 +1,21 @@
 [id='managing-business-central-githook-notifications-exit-codes-types-con']
-= Types of exit codes
+= Git hook notifications in {CENTRAL}
 
-{CENTRAL} has the following three types of exit codes:
+You can view Git hook notifications in {CENTRAL}. There are three types of notification, one for each Git hook exit code type.
```

Comment:
notifications. Need to have your numbers agree.

Reply:
Simplified paragraph and corrected Git hook notification sections.